### PR TITLE
fix: #233 remove some warnings fix probant display

### PR DESCRIPTION
**Pull request by evans-g-crsj**
# [233] [Display family position correctly]

closes #[233]

## Description
In the family tab, family position should be displayed correctlty.

Critères de succès

## Validation

- [ ] Test Coverage
- [ ] QA Done
- [ ] Design/UI Approved from design

## Screenshot

before
beforeScreenshot

after
afterscreenshot

## QA

manually test in the family tab of patient page that family position is correctly displayed. If not, pls check the data in fhir so that we can see if the issue is on the backend or the frontend.
...

### CODE REVIEW

File: client/src/components/screens/Patient/components/PatientDetails/index.tsx [hunk 1]
```diff
@@ -36,7 +35,7 @@ const ProfileCard: React.FC<{patient: ParsedPatientData}> = ({ patient }) => {
             className="link--underline"
             onClick={() => dispatch(navigateToPatientScreen(patient.familyRelation))}
           >
-            { `${parent.lastName.toUpperCase()} ${parent.firstName} (${intlDetails})` }
+            { `${parent?.lastName?.toUpperCase()} ${parent?.firstName} (${intlDetails})` }
```
Comment (evans-g-crsj) [inline client/src/components/screens/Patient/components/PatientDetails/index.tsx:38]: tempory fix

File: client/src/components/screens/Patient/components/PrescriptionsTab/components/Prescription/ClinicalSigns.tsx [hunk 2]
```diff
@@ -44,13 +44,15 @@ const ClinicalSigns: React.FC<Props> = ({ clinicalImpression, hpos }) => {
     (hpo) => get(clinicalImpression, 'investigation[0].item', []).find(
       (item: Reference) => item.reference.indexOf(hpo.id) !== -1,
     ) != null,
-  ).map((obs) => ({
+  ).map((obs, index) => ({
```
Comment (evans-g-crsj) [inline client/src/components/screens/Patient/components/PrescriptionsTab/components/Prescription/ClinicalSigns.tsx:47]: fix key warning

File: client/src/components/screens/Patient/components/PrescriptionsTab/components/Prescriptions.tsx [hunk 3]
```diff
@@ -220,7 +220,10 @@ const Prescriptions: React.FC<Props> = ({ prescriptions, clinicalImpressions })
                         <Button
                           className={`button--borderless ${!isDraft ? 'button--disabled' : ''}`}
                           icon={<DeleteOutlined />}
-                          onClick={() => alert('Feature not yey implemented')}
+                          onClick={() => {
+                            /* eslint-disable-next-line no-alert */
```
Comment (evans-g-crsj) [inline client/src/components/screens/Patient/components/PrescriptionsTab/components/Prescriptions.tsx:224]: remove warnings from console

File: client/src/components/screens/Patient/components/PrescriptionsTab/components/StatusLegend/index.tsx [hunk 4]
```diff
@@ -25,9 +26,9 @@ const StatusLegend: React.FC<Props> = ({ isVisible, onClose }) => (
     <dl className="status-legend">
       {
         statusNames.map((statusName) => (
-          <>
+          <Fragment key={statusName}>
```
Comment (evans-g-crsj) [inline client/src/components/screens/Patient/components/PrescriptionsTab/components/StatusLegend/index.tsx:29]: fix key warning

File: client/src/helpers/fhir/ObservationHelper.ts [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import get from 'lodash/get';
```
Comment (evans-g-crsj) [inline client/src/helpers/fhir/ObservationHelper.ts:1]: moved file to start extract things

File: client/src/store/FamilyMemberTypes.ts [hunk 6]
```diff
@@ -0,0 +1,36 @@
+import { GroupMemberStatusCode } from '../helpers/fhir/patientHelper';
```
Comment (evans-g-crsj) [inline client/src/store/FamilyMemberTypes.ts:1]: moved some coherent elements at the same place

File: client/src/store/ObservationTypes.ts [hunk 7]
```diff
@@ -0,0 +1,11 @@
+import { Observation } from '../helpers/fhir/types';
```
Comment (evans-g-crsj) [inline client/src/store/ObservationTypes.ts:1]: moved some coherent elements at the same place

**Review by francisl — APPROVED** [review]
:shipit:

**Review by oliviercperrier — APPROVED** [review]

**Pull request merged**